src/platforms/unity/troubleshooting.mdx

```diff
@@ -6,6 +6,18 @@ description: "Learn more about how to troubleshoot common issues with the Unity
 
 ## Installation Issues
 
+### Unable to add package Sentry Unity Package from Git
```

Comment:
Two points down the list we already cover this issue. But maybe we need to highlight the actual error message in a better way.

Reply:
Suggest moving the text sample and the screenshot under the existing content and then update that text to include all of the scenarios (trailing whitespace, additional spaces or lines).

Reply:
I moved the Sample and the Screenshot under the existing content.
About the additional scenarios, they may not be necessary since Unity converts newlines to whitespace.

Reply:
Additionally, since if you add whitespace can also result in the same error, I altered the line `Confirm the URL doesn't have a trailing whitespace at the end` to
`Confirm the URL doesn't have a trailing whitespace between the URL`. Is that ok for you @imatwawana and @bitsandfoxes?